packages/osd-pm/src/utils/yarn_lock.ts

```diff
@@ -75,6 +77,33 @@ export async function readYarnLock(osd: OpenSearchDashboards): Promise<YarnLock>
   return {};
 }
 
+/**
+ * Converts relative `file:` paths to absolute paths
+ * Yarn parsing method converts all file URIs to relative paths and this
+ * breaks the single-version requirement as dependencies to the same path
+ * would differ in their URIs across OSD and packages.
+ */
+function fixFileLinks(yarnLock: YarnLock): YarnLock {
```

Comment:
Are you planning to add a unit test for the new fn in `osd-pm`?

Reply:
`yarn_lock.ts` has no unit tests and I conveniently forgot about it. Thanks for reminding me.

Reply:
Created a unit test for this.